packages/utils/plugins-helpers/src/types.ts

```diff
@@ -414,6 +414,18 @@ export namespace Types {
      * For more details: https://graphql-code-generator.com/docs/getting-started/development-workflow#watch-mode
      */
     watch?: boolean | string | string[];
+    /**
+     * @description Allows overriding the behavior of watch to use stat polling over native file watching support.
+     *
+     * You can specify a boolean to toggle polling on or off (default: false) or an interval option to configure polling.
+     *
+     * For more details: https://graphql-code-generator.com/docs/getting-started/development-workflow#watch-mode
+     */
+    watchPolling?:
```

Comment:
Maybe it can be `watchConfig`? and then `{ pollingInterval: number, usePolling: boolean }` (or, same as it's defined in chokidar).